packages/lib/utils.ts

```diff
@@ -83,3 +83,5 @@ export const isLight = (color: string) => {
   }
   return r * 0.299 + g * 0.587 + b * 0.114 > 128;
 };
+
+export const isCapitalized = (str: string) => str.charAt(0) === str.charAt(0).toUpperCase();
```

Comment:
Just a suggestion, We have some util functions related to string in `lib/strings.ts`, but that too feels a bit odd. So can we move `strings.ts` to` lib/utils` folder and also include this function there. What do you think 🤔

Reply:
sounds good, this file also has some functions for colors, so maybe we can also have a `colors.ts` inside the utils dir with those functions